package.json

```diff
@@ -235,6 +249,10 @@
         }
       ],
       "commandPalette": [
+        {
+          "command": "microsoft-powerapps-portals.webExtension.init",
+          "when": "isWeb && config.powerPlatform.experimental.enableWebExtensionFeatures"
```

Comment:
We will also need the to disable the PAC UI panels in `power-platform-activitybar` and every PAC CLI related command from displaying in the Command Pallet when `isWeb`

Reply:
Done.

Reply:
Missed removing `pacCLI.openDocumentation` from the Command Pallet

Reply:
Oh I purposefully left this because I think documentation for webExtension might go here. But anyways, for now I have disabled this command as well.